test/routers/Webhooks.ts

```diff
@@ -91,3 +110,27 @@ test.serial('testing is_volunteer false', async t => {
     .send(user2);
   t.is(res.status, 400);
 });
+
+test.serial('testing empty tags', async t => {
+  sinon.stub(env, 'nationbuilderWebhookToken').value(webtoken1);
+  t.log(process.env.NATIONBUILDER_WEBHOOK_TOKEN);
+  sendgridApiClient.sendEmail = sinon.stub().returns(result1);
+  airtableApiClient.getOrganizations = sinon.stub().returns(result1.organizations);
+  const res = await request(app)
+    .post('/webhooks/nationbuilder/personCreated')
+    .send(user3);
+  t.is(res.status, 200);
+  t.deepEqual(res.body, result1);
+});
+
+test.serial('testing sendEmail error', async t => {
```

Comment:
Is the sendEmail error supposed to happen with a 200 response? Maybe you wrote the wrong description for this test?

Reply:
Oh yikes, yeah I think I know what happened. I meant to write a test for the sendEmail error and used the previous test as a template, but then ran into the issues I explained to you but forgot to remove this test. I'll quickly remove it.